QUICKSTART-ECS.md

```diff
@@ -1,6 +1,6 @@
 # Using a Bottlerocket AMI with Amazon ECS
 
-> The [ECS variant](./aws-ecs.md) of Bottlerocket is currently in a developer preview phase and we're looking for your
+> The [ECS variant](variants/aws-ecs-1/Cargo.toml) of Bottlerocket is currently in a developer preview phase and we're looking for your
```

Comment:
Hmm, I wonder if `variants/README.md#aws-ecs-1-amazon-ecs-container-instance` would be more helpful, since it's human-readable and may have more information in the future?  The Cargo.toml is pretty specific technical knowledge and this is right at the top of the ECS setup introduction.  I might even prefer having no link here at all.